LR_Tourist/TouristWebApp/Program.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace TouristWebApp
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
```

Comment:
If you use NLog - some configuration should be here:
https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-3

Reply:
fixed